### DISCUSSION TITLE
Unable customise the entire UI

### DISCUSSION THREAD

**Discussion by martian-coder** [bionic-gpt/bionic-gpt#263] [Announcements]
Have mailed my questions

**Comment by ianpurton** [comment]
The UI is implemented with Dioxus https://dioxuslabs.com/

**Reply by martian-coder** [reply]
Thank you, appreciate your quick response, I will try it out , hope you don't mind if I ask stupid questions as it's new to me. I have few use cases. I will play around with it.

**Reply by ianpurton** [reply]
No problem. Ask as many questions as you like.

**Reply by martian-coder** [reply]
Have mailed to contact email.

**Comment by martian-coder** [comment]
Can we setup a call, as I find this very interesting and would like to explore in detail as I work on python n c#, rust is my next goto lang. I will mail the details the use cases,  if you can share official mail detail.

**Reply by ianpurton** [reply]
Our contact details are here https://bionic-gpt.com/contact/

**Reply by martian-coder** [reply]
Have mailed it  thank you
